reana_client/api/client.py

```diff
@@ -499,9 +499,10 @@ def upload_to_server(workflow, paths, access_token):
     if not workflow:
         raise ValueError("Workflow name or id must be provided")
     if not paths:
-        raise ValueError(
-            "Please provide path(s) to file(s) that " "should be uploaded to workspace."
+        logging.info(
+            "No path(s) to file(s) that should be uploaded to workspace was provided."
         )
+        return
```

Comment:
I see that with the value returned by this function we then iterate [here](https://github.com/reanahub/reana-client/blob/ae5180ed53d7a3a121402ab0ffe22d772afa93e4/reana_client/cli/files.py#L299). Maybe we should return an empty list then? (haven't tested it)